src/app/file-explorer.js

```diff
@@ -20,12 +20,9 @@ var css = csjs`
     background-color  : white;
   }
   .remove             {
-    align-self        : center;
-    padding-left      : 10px;
+    float             : right;
```

Comment:
in flex you have 2 axis. One is for vertical, the other for horizontal. The layout among one axis can be changed with the property `justify-content` and the other with the `align-items` ..but which axis is which depends on the flex layout, which can be `flex-direction: row` (default) or `flex-direction: column;`
The axis (vertical/horizontal) will switch based on that

...now if you want to layout elements among the axis, you have many options... to put them in the begining or end there are the properties, e.g.: `align-items: flex-start;` or `align-items: flex-end;`  ...and the same for `justify-content: flex-start;` or `justify-content: flex-end;` ...but it depends on row/column whether that affects the x- or y-axis :-)